packages/sdks-tests/src/tests/hover-animation.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { expect } from '@playwright/test';
+import { excludeTestFor, test, isOldReactSDK, isRNSDK } from './helpers/index.js';
+
+test.describe('Hover animations', () => {
+  test.fail(excludeTestFor({ angular: true }), 'Angular Gen2 SDK not implemented.');
+  test.fail(isRNSDK); // hover animation is not handled in react native SDK
+  test('Button should change color when hovered', async ({ page }) => {
+    await page.goto('/hover-animation');
+
+    const button = isOldReactSDK ? page.locator('span') : page.locator('button');
+
+    const initialColor = await button.evaluate(el => getComputedStyle(el).backgroundColor, null, {
+      timeout: 10000,
+    });
```

Comment:
I used to get styles this way but it caused issues. Also, not a fan of all the `timeout`s in this test.

Have you tried `await expect(button).toHaveCSS(key, value)`? That's that we use everywhere now and it works OK

Reply:
Thanks Sami! Since this is a hover effect, I thought we might want to use a computed style to get the actual color. Let me use `toHaveCss` and try.

Reply:
Thanks a lot Sami! I've updated the tests to use `toHaveCSS` and removed the timeouts